kpops/pipeline/__init__.py

```diff
@@ -96,7 +96,7 @@ def generate(self) -> list[dict[str, Any]]:
         return [component.generate() for component in self.components]
 
     def to_yaml(self) -> str:
-        return yaml.dump(self.generate(), Dumper=CustomSafeDumper)
+        return yaml.dump(self.generate(), sort_keys=False, Dumper=CustomSafeDumper)
```

Comment:
How is ordering for fields other than name and type determined now?

Reply:
that's completely up to Pydantic. https://docs.pydantic.dev/latest/concepts/models/#field-ordering